test/e2e/Dockerfile.connect

```diff
@@ -1,32 +1,81 @@
 # Defines an image that installs connect-manager and a Connect DEB
 # The Connect host to be used as the publishing target.
+# Multiple versions of R, Python and Quarto are installed and configured.
 
-FROM ubuntu:22.04 AS e2e-publisher-connect
+FROM e2ebase AS connect-publisher-e2e
 
-ARG DEBIAN_FRONTEND=noninteractive
-ARG R_VERSION=4.3.0
-ARG R_REPOSITORY=https://packagemanager.posit.co/cran/__linux__/jammy/latest
-ARG PYTHON_VERSION=3.11.3
-ARG CONNECT_MANAGER_VERSION=0.5.1
 RUN export TZ=America/New_York
 
-# Install prerequisites
-RUN apt-get update && apt-get install -y \
-  curl \
-  xz-utils
+# Setup users and groups using cloned script from Connect's docker image: 
+# docker/ubuntu24/Dockerfile
+
+# An incoming Jenkins uid/gid alters how the account is created. Allows
+# matching uid/gid on the launching Jenkins agent. Configured late to keep
+# more layers reusable.
+ARG JENKINS_GID=999
+ARG JENKINS_UID=999
+
+# Create the jenkins user with the same id:gid as the jenkins-node
```

Comment:
Is this setup of jenkins, users and groups really needed for Publisher tests?

Reply:
I matched what was being done within the Connect dev image, so that the code would run the same. There are certain scenarios in which we would fail to deploy because of content ownership and collaboration. So between these two factors, as well as making it easier to maintain (IMO - when different than connect's dev image, update it), I think there is an advantage to not reducing these small items away.

Reply:
Got it, I see that it could be tempting to have the same setup that we have in Connect dev image, still I'll push a bit back on this because:
1. We would need to remember to update in both places, easy to get out of sync
2. Putting unnecessary effort on syncing something that potentially is not needed

I think there is a fine line between having e2e tests for `Publisher + Connect` vs e2e tests publishing to Connect. There are tests that should be exclusive to Connect. 

What you have in `Dockerfile.base` is great and totally needed to test requirements between Publisher and Connect, aside from that, I don't think having things like groups and RunAs specifics is needed, a few users should be enough to test publishing scenarios including permissions and collaboration.